webui/react/src/types.ts

```diff
@@ -769,14 +769,25 @@ export const ExperimentItem = t.intersection([
     userId: t.number,
   }),
 ]);
-export type ExperimentItem = t.TypeOf<typeof ExperimentItem>;
+export type BulkExperimentItem = t.TypeOf<typeof BulkExperimentItem>;
+
+export const FullExperimentItem = t.intersection([
+  BulkExperimentItem,
+  t.partial({
+    config: ExperimentConfig,
+  }),
```

Comment:
i know this is inherited, but we check for `config` when determining whether or not a task is an experiment in `isExperiment` -- can we depend on it being here or not?

Reply:
good question: i think we should use `hyperparameters` instead of `config`. 
`isExperiment` isnt used anywhere fyi

```ts
export const isExperiment = <T extends BulkExperimentItem | FullExperimentItem>(
  obj: AnyTask | T,
): obj is T => {
  return 'hyperparameters' in obj && 'archived' in obj;
};
```